module/parser/src/commonMain/kotlin/org/cru/godtools/shared/tool/parser/ManifestParser.kt

```diff
@@ -8,7 +8,8 @@ import org.cru.godtools.shared.tool.parser.xml.XmlPullParserException
 import org.cru.godtools.shared.tool.parser.xml.XmlPullParserFactory
 
 open class ManifestParser(private val parserFactory: XmlPullParserFactory, val defaultConfig: ParserConfig) {
-    suspend fun parseManifest(fileName: String, config: ParserConfig = defaultConfig): ParserResult = try {
+    suspend fun parseManifest(fileName: String) = parseManifest(fileName, defaultConfig)
```

Comment:
left a comment on your actual PR that you actually didn't need this method due to how you were using `IosManifestParser`